src/main/java/org/folio/des/service/impl/JobServiceImpl.java

```diff
@@ -136,6 +140,19 @@ public org.folio.des.domain.dto.Job upsertAndSendToKafka(org.folio.des.domain.dt
       result.setExitStatus(ExitStatus.UNKNOWN);
     }
 
+    Optional.ofNullable(jobDto.getExportTypeSpecificParameters())
+      .map(ExportTypeSpecificParameters::getVendorEdiOrdersExportConfig)
+        .map(VendorEdiOrdersExportConfig::getExportConfigId).ifPresent(configId -> {
+          try {
+            client.getConfigById(configId.toString());
+          } catch (NotFoundException e) {
+            log.info("Config with id {} not found", configId.toString());
+            jobDto.getExportTypeSpecificParameters().getVendorEdiOrdersExportConfig().getEdiSchedule().
+            getScheduleParameters().setSchedulePeriod(ScheduleParameters.SchedulePeriodEnum.NONE);
+
+            throw new NotFoundException(String.format(INTEGRATION_NOT_AVAILABLE, configId));
```

Comment:
Basically you create new NotFoundException and lost context of previous exception,
so you can replace to: throw e;

Reply:
done!